src/docs-style-guide.rst

```diff
@@ -25,7 +25,28 @@ Whenever U.S. English and British (or other) English spelling or usage disagree,
 .. code-block:: rst
 
   The color of the button is gray.
+
+.. code-block:: python
```

Comment:
Let's add a [+ Show Code]/[ + Hide Code] JS toggle to these blocks. 
You can use my code for the screenshots as a jumping off point.